apps/examples/stack_protection/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_STACK_PROTECTION
+	bool "Stack Protection TestCase Example"
```

Comment:
It does not have the same format with another testcase which is in tc folder.
Let's say `Stack Protection Test example`

Reply:
added to example test case for support in tash and non tash case, let me know if that is fine.